apps/semver-yeasy/semver-yeasy.sh

```
@@ -189,23 +189,87 @@ calculate-version)
-        -H "Content-Type: application/json" \
-        -H "Authorization: token ${GITHUB_TOKEN}" \
-        "https://api.github.com/repos/$GITHUB_REPOSITORY/pulls/$PR_NUMBER"
+    # echo "$PR_DESCRIPTION" | wc -l >> $GITHUB_OUTPUT
+    # echo "UPDATED_PR_BODY=${UPDATED_PR_BODY}\\n" >> $GITHUB_OUTPUT
+    # UPDATED_PR_BODY+="$line\\n"
+    echo "UPDATED_PR_BODY=${UPDATED_PR_BODY}\\n" >> $GITHUB_OUTPUT
+
+    # Only update the PR if PR_DESCRIPTION was not empty (i.e., not a unit test)
+    if [[ -z "${RUN_ENV}" ]]; then
+        # Update the PR with the updated description
+        ${JQ_EXEC_PATH} -nc --arg body $(echo "$UPDATED_PR_BODY" | sed 's/\\n/\n/g') '{"body": $body}' | \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)